package.py

```diff
@@ -231,10 +237,12 @@ def main():
     # Z3
     flush("* Finding and downloading Z3 releases")
     releases = list(discover(args.version))
```

Comment:
If this is Z3 releases, let's use a variable name like z3releases.
Same for 'version'
And there are other comments and names that are ambiguous between Dafny and Z3

Reply:
It's a collection of objects of the class `Release` defined in package.py, corresponding both with Z3 artefacts and Dafny artefacts, so I think the name might already be good. Also, since this PR didn't introduce that name I'd prefer to leave it.